lib/nimble_template/addons/variants/phoenix/api/params_validation.ex

```diff
@@ -3,7 +3,43 @@ defmodule NimbleTemplate.Addons.Phoenix.Api.ParamsValidation do
 
   use NimbleTemplate.Addons.Addon
 
-  def do_apply(%Project{} = project, _opts), do: copy_files(project)
+  def do_apply(%Project{} = project, _opts) do
+    project
+    |> copy_files()
+    |> edit_files()
+  end
+
+  defp edit_files(project) do
+    edit_web_entrypoint(project)
+  end
+
+  defp edit_web_entrypoint(%Project{web_module: web_module, web_path: web_entrypoint} = project) do
```

Comment:
O think the corrct name is [`entry point`](https://en.wikipedia.org/wiki/Entry_point)
```suggestion
  defp edit_web_entry_point(%Project{web_module: web_module, web_path: web_entrypoint} = project) do
```